avalon/vendor/toml.py

```diff
@@ -4,9 +4,11 @@
 import datetime
 import io
 
+
```

Comment:
Best not to edit vendor files, we can rely on them being tested in their own project and CI pipeline. Any edits we make would be overwritten whenever we update the library anyway.

Was it throwing errors on Travis? They should be excluded from our tests.